### PR TITLE
Judge

### PR DESCRIPTION
**Pull request by AlannaFey**
Added three functions to the backend.  judgeIsOnline to ping the judge, submit to submit problems, and get_verdict to get the information regarding the submission.

### Judge online example
image

### Submit example
image

### Get Verdict Example
image

### CODE REVIEW

File: functions/tsconfig.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "sourceMap": true,
     "strict": true,
     "target": "es2017",
-    "esModuleInterop": true,
+    "esModuleInterop": true
```
Comment (cameroncuster) [inline functions/tsconfig.json:10]: is this a prettier thing?

File: functions/src/judge.ts [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      return res.status(500).json({ error: err })
+    })
+  return
+}
+
+export async function get_verdict(req: Request, res: Response) {
+  // TODO: After the submission is no longer pending, delete it :)
+  let submission_id: string = req.body.token
+  await getDoc(doc(db, 'Submissions', submission_id))
+    .then(async (a) => {
```
Comment (cameroncuster) [inline functions/src/judge.ts]: what is `a`? can we use a different name

File: functions/src/index.ts [hunk 3]
```diff
@@ -45,6 +45,7 @@ app.post('/helloWorld', (req: Request, res: Response) => {
 
 import { emailLogin, emailRegister } from './user'
 import { getUserData, updateUserData } from './userData'
+import { get_verdict, judgeIsOnline, submit } from './judge'
```
Comment (cameroncuster) [inline functions/src/index.ts]: super nit: mixing camelCase and snake_case is funny

**Review by cameroncuster — APPROVED** [review]
So I didn't take a super close look at all of this, but I think the best course of action is to move forward and just have us test by using it. If and when we run into bugs we can fix them then. It's probably too much work to set up any integration or E2E testing at this stage of the project. I also don't have a ton of context on everything we still need to do, so I'm just approving to give the freedom for you to merge when you're ready. LMK if you want me to take a second look at anything.

File: functions/tsconfig.json (hunk 1 above)
Comment (AlannaFey) [reply functions/tsconfig.json:10]: probably, i just ran prettier on everything

File: functions/src/judge.ts (hunk 2 above)
Comment (AlannaFey) [reply functions/src/judge.ts]: it is the submission document in the database, I can rename it to submission

File: functions/src/index.ts (hunk 3 above)
Comment (AlannaFey) [reply functions/src/index.ts]: oops lol yea i can fix that

**Pull request merged**